modules/loki/main.tf

```diff
@@ -1,11 +1,162 @@
 locals {
-  chart_name    = "loki-stack"
-  chart_version = var.chart_version
-  release_name  = "loki"
-  namespace     = var.namespace
-  repository    = "https://grafana.github.io/loki/charts"
+  chart_name     = "loki-stack"
+  chart_version  = var.chart_version
+  release_name   = "loki"
+  namespace      = var.namespace
+  repository     = "https://grafana.github.io/loki/charts"
+  provider_url   = replace(var.oidc_provider_issuer_url, "https://", "")
+  bucket_name    = "${var.cluster_name}-loki-${data.aws_caller_identity.loki.account_id}"
+  dynamodb_table = "${var.cluster_name}-loki"
```

Comment:
I think Loki 1.5 shipped with a new indexing mechanism which removes the need for DynamoDB. I haven't looked into the pros and cons of it. Have you? Otherwise it might be something to consider.

Reply:
It's still beta